javaslang/src/main/java/javaslang/collection/AbstractMultimap.java

```diff
@@ -229,8 +231,8 @@ public M filter(Predicate<? super Tuple2<K, V>> predicate) {
         return foldLeft(HashMap.empty(), (map, entry) -> {
             final C key = classifier.apply(entry);
             final Multimap<K, V> values = map.get(key)
-                    .map(entries -> entries.put(entry._1, entry._2))
-                    .getOrElse(createFromEntries(Iterator.of(entry)));
+                                             .map(entries -> entries.put(entry._1, entry._2))
+                                             .getOrElse(createFromEntries(Iterator.of(entry)));
```

Comment:
Done, pushed a new version, squashing the changes (as the rename isn't separated anymore).